sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -113,18 +99,22 @@ private LocalDocumentStorage(DatabaseManager databaseManager) {
     }
 
     <T> void write(Document<T> document, WriteOptions writeOptions) {
-        AppCenterLog.debug(LOG_TAG, String.format("Trying to replace %s:%s document to cache", document.getPartition(), document.getId()));
-        Calendar now = Calendar.getInstance();
-        Calendar expiresAt = now;
+        write(document, writeOptions, PENDING_OPERATION_CREATE_VALUE);
+    }
+
+    <T> void write(Document<T> document, WriteOptions writeOptions, String pendingOperationValue) {
+        AppCenterLog.debug(LOG_TAG, String.format("Trying to write %s:%s document to cache", document.getPartition(), document.getId()));
```

Comment:
It is definitely a replace as in upsert, why renaming log? We are calling replace at the end of the method.